sites/packworld/server/templates/index.marko

```diff
@@ -43,20 +67,98 @@ $ const { id, alias, name, pageNode } = data;
         />
       </div>
       <div class="col-lg-4 mb-block page-rail">
-        <marko-web-gam-display-ad id="gpt-ad-rail2" />
+        <div id="home-02-right" />
+        <div id="home-03-right" />
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-lg-4 mb-block">
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content"
+          params={ sectionAlias: "machinery", limit: 4, queryFragment }
+        >
+          <website-content-list-flow nodes=nodes>
+            <@header>
+              <marko-web-link href="/machinery">Machinery</marko-web-link>
+            </@header>
+          </website-content-list-flow>
+        </marko-web-query>
+      </div>
+      <div class="col-lg-4 mb-block">
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content"
+          params={ sectionAlias: "design", limit: 4, queryFragment }
+        >
+          <website-content-list-flow nodes=nodes>
+            <@header>
+              <marko-web-link href="/design">Design</marko-web-link>
+            </@header>
+          </website-content-list-flow>
+        </marko-web-query>
+      </div>
+      <div class="col-lg-4 mb-block">
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content"
+          params={ sectionAlias: "issues", limit: 4, queryFragment }
+        >
+          <website-content-list-flow nodes=nodes>
+            <@header>
+              <marko-web-link href="/issues">Issues</marko-web-link>
+            </@header>
+          </website-content-list-flow>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-lg-4 mb-block page-rail">
+        <div class="ad-container ad-container--in-card">
+          <div id="home-04-tall-left" />
+        </div>
+      </div>
+      <div class="col-lg-8 mb-block">
+        <marko-web-query|{ nodes }|
+          name="all-published-content"
+          params={ contentTypes: ["Video"], limit: 5, queryFragment }
+        >
+          <website-content-list-flow nodes=nodes>
+            <@header>
+              Video
+            </@header>
+          </website-content-list-flow>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <marko-web-query|{ nodes }|
+      name="website-optioned-content"
+      params={ sectionAlias: "home", optionName: "Pinned", limit: 5, skip: 9, requiresImage: true, queryFragment }
+    >
+      <website-content-card-deck-flow nodes=nodes ad-index=1 ad-name="imu2" >
+        <@native-x index=2 name="load-more" />
+      </website-content-card-deck-flow>
+    </marko-web-query>
+
+    <div class="row">
+      <div class="col-lg-12 mb-block page-rail">
+        <div class="ad-container ad-container--in-card">
+          <div id="home-bottom-wide" />
+        </div>
```

Comment:
There's no need to add the `ad-container` wrapper (vertical align) or `page-rail` (horizontal align) divs, as this is full width. Just wrap the custom div with `.row > .col-lg-12.mb-block`.